house_robber_brute_force.py

```diff
@@ -0,0 +1,33 @@
+# Time Complexity : Add - O(2^n)
+# Space Complexity :O(log n)
+# Did this code successfully run on Leetcode : No. It was a brute force technique so time exceeded.
+
+# Any problem you faced while coding this : A little bit, I am weak in Recursion
```

Comment:
No worries as of now, you'll learn more about recursion soon.
Try to solve the solution taught in class and reattempt it to solve again